k8s/istio-install/istio-manager.yaml

```diff
@@ -2,26 +2,26 @@
 apiVersion: v1
```

Comment:
Controlplane has overloaded semantic. In fact, we install control plane and management plane. Plus, this directory is just about installation.